recipes/sagemaker-code-editor/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "sagemaker-code-editor" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://sagemaker-code-editor-1696292125.s3.us-west-2.amazonaws.com/code-editor.tar.gz  # [linux64]
+  sha256: 9effda827c45e1fd8dfc534ce310791ee46070d818ede89abf19e3ec9c8906d0  # [linux64]
+  folder: sagemaker-code-editor
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - sysroot_linux-64 ==2.17
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - pkg-config
+    - nodejs ==18.18.2
+    - openssl ==3.1.4
+    - jq
+    - krb5 ==1.21.2  # [linux]
+    - libsecret  # [linux]
+    # node-gyp is broken with python 3.12 - https://github.com/nodejs/node-gyp/issues/2869, so pulling in distutils
+    - setuptools
+    - yarn <2
+  host:
+    - nodejs ==18.18.2
```

Comment:
nodejs is part of the [global pinnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/4cea2512b45b87086db6251e12c7bc233c53468c/recipe/conda_build_config.yaml#L629-L631), so this is effectively opting out of the migration system and the ABI compatibilities. Can you elaborate on why this specific version is required?

Reply:
To be able to build the (modified) open source Code - [OSS repo](https://github.com/microsoft/vscode), we need `Node.JS version >=18.15.x and <19`
Also, it was set to a specific version since we had to install it in a conda env where the original version of nodejs was frozen to 18.18.2. So, if we didn't specify the same version here, installation would fail.
However, we should not have that restriction anymore, so removing.